config/locales/de/cms.yml

```diff
@@ -0,0 +1,466 @@
+            title: "Auf dem neuesten Stand"
+            description: "Wenn ein Ereignis erstellt oder aktualisiert wird"
+          should_need_review:
+            icon: orange question circle # Do not translate
+            active_label: "überprüfung notwendig"
+            inactive_label: "benütigt in kürze Überprpfung"
+            title: "überprüfung notwendig"
+            description: "Wenn %{duration} vergangen ist ohne das die Veranstalltung überprüft wurde"
+          should_expire:
+            icon: red times circle # Do not translate
+            active_label: "Abgelaufen"
+            inactive_label: "Abgelaufen"
+            label: "läuft ab" # wie in  "Die Veranstaltung läuft am 3. März ab"
+            title: "Abgelaufen"
+            description: "Wenn ein Ereignis nicht in %{duration} aktualisiert wurde"
+      registration:
+        title: "Warum sollten Sucher sich auf Sahaj Atlas anmelden?"
+        description:
+          introduction: "Auch wenn es möglich ist das sich Sucher auf externen Websiten (Meetup, Eventbrite, Facebook usw.) anmelden, gibt es mehre vorteile, wenn sich Sucher direct auf dem Sahaj Atlas registrieren."
+        benefits:
+          accessibility:
+            icon: universal access # Do not translate
+            title: "Zugänglichkeit"
+            description: "Viele Menschen haben keine Facebook- oder Meetup-Konten. Die Popularität von Facebook ist bei der jüngeren Generationen stark reduziert, und Meetup ist bei älteren Menschen weniger beliebt. Die Registrierung direkt auf Sahaj Atlas ist einfacher und erfordert kein Konto, was die warscheinlichkeit erhöht das sich sucher anmelden."
+          reminders:
+            icon: bell # Do not translate
+            title: "Automatische Erinnerungen"
+            description: "Sucher, die sich auf Sahaj Atlas anmelden, erhalten eine Bestätigungs-E-Mail und eine automatische Erinnerung per E-Mail 1 Tag bevor ihrer erster Kurs beginnt. Der Yogi(ni), der/die das öffentliche Programm ausführt, wird jede Woche eine Liste neuer Anmeldungen per E-Mail erhalten."
+          marketing:
+            icon: envelope # Do not translate
+            title: "Future Marketing"
+            description: "Wenn sich Sucher auf Sahaj Atlas anmelden, können wir ihre E-Mail-Adresse (ohne die Einschränkungen von Meetup, Eventbrite oder Facebook) speichern. Dies bedeutet, dass wir diese E-Mail-Adressen verwenden können, um Folgeveranstaltungen vorzuschlagen oder spezielle Sahaj-Ereignisse in ihrer Gegend zu empfehlen (zB.: Konzerte oder Zwischenprogramme)."
+      contact:
+        title: "Ich habe andere Fragen oder Vorschläge"
+        description: "Wenn Sie weitere Hilfe benötigen oder wenn Sie mit der Entwicklung des Sahaj Atlas helfen möchten, schreiben Sie bitte an die unten angegebene E-Mail. Bitte schreiben Sie Ihre Nachricht in Englisch."
+
+    country_codes:
+      gb: "UK"
+      us: "USA"
+      de: "Deutschland"
+      at: "Österreich"
```

Comment:
This section should contain abbreviated versions of the country names. Germany and Austria will already use AT and DE by default, but if there is an abbreviation for a country which is more recognisable to German speakers than the ISO standard (like UK is more recognisable than GB), then it can be overridden here.

Basically I suspect you don't need either the `de` or `at` entry here.